KsApi/combine/CombineTestObserver.swift

```diff
@@ -1,13 +1,17 @@
 import Combine
 import Foundation
+// TODO: do I need to move CombineTestObserver into another package, so this isn't imported into KsApi?
```

Comment:
I forget whether or not Swift is smart enough not to link a framework added to a file, if that file isn't actually added to a target. We only use `CombineTestObserver` in test packages.

We may need to move this file into its own package.